topology/probes/k8s/networkpolicy.go

```diff
@@ -241,83 +235,63 @@ func getFieldPorts(np *v1beta1.NetworkPolicy, ty PolicyType) string {
 }
 
 func (npl *networkPolicyLinker) newEdgeMetadata(ty PolicyType, target PolicyTarget, point PolicyPoint) graph.Metadata {
-	m := NewEdgeMetadata(Manager, "association")
-	m.SetField("RelationType", "networkpolicy")
+	m := NewEdgeMetadata(Manager, "networkpolicy")
 	m.SetField("PolicyType", string(ty))
 	m.SetField("PolicyTarget", string(target))
 	m.SetField("PolicyPoint", string(point))
 	return m
 }
 
-func (npl *networkPolicyLinker) createLinks(np *v1beta1.NetworkPolicy, npNode, filterNode *graph.Node, ty PolicyType, target PolicyTarget, point PolicyPoint, pods []metav1.Object) (edges []*graph.Edge) {
+func (npl *networkPolicyLinker) create1SideLinks(np *v1beta1.NetworkPolicy, npNode, filterNode *graph.Node, ty PolicyType, target PolicyTarget, point PolicyPoint, pods []metav1.Object) (edges []*graph.Edge) {
 	podNodes := objectsToNodes(npl.graph, pods)
-	metadata := npl.newEdgeMetadata(ty, target, point)
-	for _, objNode := range podNodes {
-		if filterNode == nil || filterNode.ID == objNode.ID {
-			metadata.SetFieldAndNormalize("PolicyPorts", getFieldPorts(np, ty))
-			fields := []string{string(npNode.ID), string(objNode.ID)}
-			for k, v := range metadata {
+	m := npl.newEdgeMetadata(ty, target, point)
```

Comment:
Please avoid renaming local variables. It brings noise and makes the review harder

Reply:
ok reverted `m` to `metadata`